doc/sphinx/language/core/conversion.rst

```diff
@@ -3,11 +3,75 @@
 Conversion rules
 ----------------
 
-|CiC| has conversion rules that can be used to determine if two
-terms are equal by definition, or :term:`convertible`.
-Conversion rules consist of reduction rules and expansion rules.
-See :ref:`applyingconversionrules`,
-which describes tactics that apply these conversion rules.
+Conversion rules convert terms to other terms that are equivalent by
+definition under the rules of |CiC|.  Conversion rules can be used, for example,
+to determine if two terms are equal by definition, or :term:`convertible`, by
+converting both terms to a normal form, then verifying they are syntactically
+equal (ignoring differences in the names of bound variables
+:term:`alpha-convertibility <alpha-convertible>`).
+
+Conversion rules can be reductions or expansions.
+Reductions convert terms to something that is incrementally closer to its
+normal form.  For example, :term:`zeta-reduction` removes
+:n:`let @ident := @term__1 in @term__2` constructs from a term by replacing
+:n:`@ident` with :n:`@term__1` wherever :n:`@ident` appears in :n:`@term__2`.
+The resulting term may be longer or shorter than the original.
+
+.. coqtop:: all
+
+   Eval cbv zeta in let i := 1 in i + i.
+
+Expansions are reductions applied in the opposite direction,
+for example expanding `2 + 2` to `let i := 2 in i + i`.  While applying
+reductions (except for eta-reduction) gives a unique result , the associated
+expansion may not give a unique result.  For example, `2 + 2` could also be
+expanded to `let i := 2 in i + 2`.  Expansions that give a unique
+result are also referred to as *contractions*.
```

Comment:
This isn't right; contraction is the inverse of expansion.  Reductions which are both guaranteed to always make the term smaller and which have a unique inverse expansion are called contractions.  (I'm not sure if both these conditions are necessary, though.). Perhaps it would be safer to call out eta contraction by name?

Reply:
Fixed the wording.

From your comment above, I thought eta-reduction was not useful in Coq.  Also IIUC not used to determine convertibility nor available in `cbv` et al.  If so, not worth mentioning.

For this chapter, I will be satisfied if I can give a reasonable concrete description of how the conversion rules rewrite terms without contradicting the theory behind them.  Hopefully explaining a bit about why they're useful, too.  Explaining how the theory works, no, not at this time.